src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -401,29 +592,36 @@ def set_datamover_params(self,
 
         """
         if self.tool == "DCP":
-            self.set_dcp_params(src_type, src_path, src_pool, src_cont,
-                                dst_type, dst_path, dst_pool, dst_cont)
+            self._set_dcp_params(src_type, src_path, src_pool, src_cont,
+                                 dst_type, dst_path, dst_pool, dst_cont)
         elif self.tool == "DSYNC":
-            self.set_dsync_params(src_type, src_path, src_pool, src_cont,
-                                  dst_type, dst_path, dst_pool, dst_cont)
+            self._set_dsync_params(src_type, src_path, src_pool, src_cont,
+                                   dst_type, dst_path, dst_pool, dst_cont)
+        elif self.tool == "DSERIAL":
```

Comment:
@daltonbohning Just wondering if there is a reason for using "dserialize" vs. "deserialize"?

Reply:
@dsikich I chose to use `dserialize` and `ddeserialize` instead of `daos-serialize` and `daos-deserialize` to avoid future name conflicts with `daos cont serialize` and `daos cont deserialize`.
Basically, the mpifileutils tools use a `d` prefix, and the daos tools use a `daos` prefix. Longer term, I've considered just using an `mfu` prefix for the mpifileutils versions to be more clear.